samples/CommunityToolkit.Maui.Markup.Sample/AppShell.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using CommunityToolkit.Maui.Markup.Sample.Constants;
+using CommunityToolkit.Maui.Markup.Sample.Pages;
+using CommunityToolkit.Maui.Markup.Sample.Pages.Base;
+using CommunityToolkit.Maui.Markup.Sample.ViewModels;
+using CommunityToolkit.Maui.Markup.Sample.ViewModels.Base;
+
+namespace CommunityToolkit.Maui.Markup.Sample;
+
+class AppShell : Shell
+{
+	readonly static IReadOnlyDictionary<Type, string> pageRouteMappingDictionary = new Dictionary<Type, string>(new[]
+	{
+		CreateRoutePageMapping<NewsPage, NewsViewModel>(),
+		CreateRoutePageMapping<SettingsPage, SettingsViewModel>(),
+		CreateRoutePageMapping<NewsDetailPage, NewsDetailViewModel>(),
+	});
+
+	public AppShell(NewsPage newsPage)
+	{
+		Items.Add(new ShellContent
+		{
+			Title = newsPage.Title,
+			Content = newsPage
+		});
+
+		//  BarBackgroundColor = ColorConstants.NavigationBarBackgroundColor;
+		//	BarTextColor = ColorConstants.NavigationBarTextColor
```

Comment:
Do we need these comments here?

Reply:
Good catch! We sure don't!

I forgot to remove those after my debugging.